spec/index.bs

```diff
@@ -3634,6 +3637,23 @@ All [=depth-or-stencil formats=] are renderable.
     1. Return `null`.
 </div>
 
+Use of some texture formats require a feature to be enabled on the {{GPUDevice}}. Because support
+for these formats may not be implemented on all systems, the associated enums may not be present.
+In order to normalize behavior across implementations attempting to use a format that requires a
+feature when the associated feature is not enabled on the device will throw an exception, making the
+behavior the same as if an undefined enum was used.
+
+See [[#texture-format-caps]] for information about which {{GPUTextureFormat}}s require features.
+
+<div algorithm>
+    <dfn abstract-op>Validate feature texture format usage</dfn> of a {{GPUTextureFormat}} |format|
```

Comment:
nit: "usage" is confusing because there are also texture usages. Maybe "Validate required features of a texture format"?

Reply:
Opted for "Validate texture format required features".